doc/1/controllers/realtime/subscribe/snippets/document-notifications-leave-scope.cs

```diff
@@ -0,0 +1,39 @@
+NotificationHandler listener = (notification) => {
+  string scope = notification.Scope.ToString();
+  if (scope == "out") {
+    Console.WriteLine($"Document moved {scope} from the scope");
```

Comment:
```suggestion
    Console.WriteLine($"Document moved {scope} of the scope");
```

Also, you don't template litterals here:

```suggestion
    Console.WriteLine("Document left the scope");
```
(and "Document entered the scope" when scope is `in`)